src/charactersheet/viewmodels/character/spells/index.html

```diff
@@ -1,659 +1,231 @@
-         aria-labelledby="addSpellLabel"
-         data-bind="modal: { onopen: modalFinishedOpening,
-          open: addModalOpen,
-          onclose: closeAddModal}">
-      <div class="modal-dialog" role="document">
-        <div class="modal-content">
-          <div class="modal-header">
-            <button type="button"
-                    class="close"
-                    data-dismiss="modal"
-                    aria-label="Close">
-              <span aria-hidden="true">&times;</span>
-            </button>
-            <h4 class="modal-title" id="addSpellLabel">Add a Spell to your Spellbook.</h4>
-        </div>
-          <div class="modal-body">
-            <form class="form-horizontal" data-bind="element: _addForm, validate: validation">
-              <div class="form-group ui-front">
-                <label for="name"
-                       class="col-sm-3 control-label">
-                        Name <i class="required"></i>
-                      </label>
-                <div class="col-sm-9">
-                  <input type="text"
-                         class="form-control"
-                         name="name"
-                         id="spellsAddNameInput"
-                         placeholder="Magic Missile"
-                         data-bind="textInput: blankSpell().name,
-                          autocomplete: { source: spellsPrePopFilter,
-                          onselect: populateSpell },
-                          hasFocus: firstModalElementHasFocus">
+  <table class="listing table table-responsive table-ac-bordered table-hover">
+    <thead>
+      <tr class="ac-table-header">
+        <th class="col-xs-1 checkbox-data-cell"
+            data-bind="click: $component.toggleSpellFilter">
+          <input type="checkbox"
+              class="filter-checkbox sr-only"
+              data-bind="checked: filteredByCastable"></input>
+          <label for="filterByCastable"></label>
+        </th>
+        <th data-bind="click: ()=> {sortBy('name');}"> Spell <span data-bind="css: sortArrow('name')"></span>
+        </th>
+        <th class="ac-table-data-cell"
+            data-bind="click: () => {sortBy('level');}"> Level <span data-bind="css: sortArrow('level')"></span>
+        </th>
+        <th class="hidden-xs"
+            data-bind="click: () => {sortBy('typeLabel');}"> Type <span data-bind="css: sortArrow('typeLabel')"></span>
+        </th>
+        <th class="hidden-xs hidden-sm"
+            data-bind="click: () => {sortBy('damageType');}"> Damage <span data-bind="css: sortArrow('damageType')"></span>
+        </th>
+        <th class="hidden-xs"
+            data-bind="click: () => {sortBy('castingTime');}"> Casting Time <span data-bind="css: sortArrow('castingTime')"></span>
+        </th>
+        <th class="hidden-xs hidden-sm"
+            data-bind="click: () => {sortBy('range');}"> Range <span data-bind="css: sortArrow('range')"></span>
+        </th>
+        <th style="col-xs-1 text-right"
```

Comment:
Should the stuff in `style` actually be in `class`? I think it's been done a couple times in this html

Reply:
nice catch. My mistake.Fixed